.github/workflows/build.yml

```diff
@@ -65,6 +69,14 @@ jobs:
         image: rabbitmq:latest
         ports: [5672/tcp]
         options: --health-cmd "rabbitmqctl node_health_check" --health-interval 10s --health-timeout 5s --health-retries 5
+      redis:
+        image: bitnami/redis:6.2
+        ports:
+          - 16379:16379
+        options: --health-cmd "redis-cli -p 16379 ping" --health-interval 10s --health-timeout 5s --health-retries 5
+        env:
+          ALLOW_EMPTY_PASSWORD: yes
+          REDIS_PORT_NUMBER: 16379
```

Comment:
This shall supersede the fork-exec from `spec/support/redis.rb`?

Reply:
Nope. The fork-exec stuff is needed for running the tests locally. (Right now, it fails silently on GitHub Actions, because there's no local `redis-server` binary.)

Reply:
Well, not that silent ;-)

https://github.com/mnemosyne-mon/mnemosyne-ruby/runs/3409058808?check_suite_focus=true#step:7:47

Reply:
Why did you prefer Redis being started as a service in Github CI to `apt install -y redis-server`?

Reply:
"prefer", haha. We used that approach for RabbitMQ and it was the "best practice" from the GitHub Actions documentation. And used similarly by the OpenTelemetry project. Would you prefer installing via APT.

Reply:
I am still thinking about it. Using `apt install redis-server` and the hook and config from spec/support would mean that even more code (the support code and config) is tested too and the local and CI environment divert even less. 🤔

Reply:
Yeah, that's also the one advantage I would see.

Reply:
As agreed in another medium, I now used APT. Slightly slower, but at least we "test" our local-dev setup code this way.